serde_v8/src/magic/buffer.rs

```diff
@@ -32,6 +34,12 @@ impl Clone for MagicBuffer {
   }
 }
 
+impl Default for MagicBuffer {
+  fn default() -> Self {
+    MagicBuffer::ToV8(Cell::new(Some(vec![0_u8; 0].into_boxed_slice())))
+  }
+}
```

Comment:
I don't agree that the Default MagicBuffer should be an empty buffer like this.

I think it makes more sense if an empty buffer is explicitly constructed in op_worker_get_message() (runtime/ops/web_worker.rs:57). That is the only place `MagicBuffer::default()` is used.